### PR TITLE
update kusion cmd docs and related codes and scripts

### LINKED ISSUE #387: Enhanced support for i18n
### What would you like to be added?

The user guide for support of i18n. When a developer updates the code using i18n, there should be a guide showing how to update the translations.

### Why is this needed?

The old user guide `pkg/util/i18n/translations/README.md` misses lots of necessary information, developer cannot use it to update translations.
And there are lots of nonstandard usages of i18n, because of the lack of i18n usage convention.
Besides, the translations of kusion commands are out of date.

### PR DESCRIPTION
**Pull request by liu-hm19**
#### What type of PR is this?

/kind documentation
#### What this PR does / why we need it:
This PR updates documentations about kusion command line and related codes and scripts. 
#### Which issue(s) this PR fixes:

Fixes #387 

#### Does this PR introduce a user-facing change?

```release-note

```
NONE
#### Additional documentation e.g., design docs, usage docs, etc.:

```docs

```

### CODE REVIEW

File: docs/cmd/en_US/kusion_apply.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## kusion apply
 
-Apply a configuration stack to resource(s) by work directory
+Apply the operation intents of various resources to multiple runtime
```
Comment (SparkYuan) [inline docs/cmd/en_US/kusion_apply.md:3]: runtime -> runtimes

File: pkg/cmd/preview/preview.go [hunk 2]
```diff
@@ -62,7 +67,7 @@ func (o *PreviewOptions) AddPreviewFlags(cmd *cobra.Command) {
 		i18n.T("Automatically show plan details with interactive options"))
 	cmd.Flags().BoolVarP(&o.All, "all", "a", false,
 		i18n.T("Automatically show all plan details, combined use with flag `--detail`"))
-	cmd.Flags().BoolVarP(&o.NoStyle, "no-style", "", false,
+	cmd.Flags().BoolVarP(&o.NoStyle, "no-style", "s", false,
```
Comment (SparkYuan) [inline pkg/cmd/preview/preview.go:70]: Using "s" as an abbreviation for "no-style" is confusing.

Comment (liu-hm19) [reply pkg/cmd/preview/preview.go]: That's true. I have deleted the short hand letter for "no-style" flag and updated the examples in relevant docs and codes.

**Review by SparkYuan — APPROVED** [review]
lgtm

**Pull request merged**